src/module/PhotonOutput1D.cpp

```diff
@@ -18,16 +18,16 @@ using namespace std;
 namespace crpropa {
 
 PhotonOutput1D::PhotonOutput1D() : out(&std::cout) {
-	KISS_LOG_WARNING << "PhotonOutput1d is deprecated and will be removed in the future. Replace with TextOutput or HDF5Output with features ObserverNucleusVeto + ObserverDetectAll";
+	KISS_LOG_WARNING << "PhotonOutput1D is deprecated and will be removed in the future. Replace with TextOutput or HDF5Output with features ObserverNucleusVeto + PhotonOutput1D";
```

Comment:
Wouldn't you still need the ObserverDetectAll?

Reply:
No, something like this would work now:

```
# observer
obs1 = Observer() # proton output
obs1.add( ObserverPoint() )
obs1.add( ObserverPhotonVeto() ) # we don't want photons here
obs1.onDetection( TextOutput('proton_output.txt', Output.Event1D) )
m.add(obs1)
obs2 = Observer() # photon output
obs2.add( ObserverPoint() )
obs2.add( ObserverNucleusVeto() ) # we don't want hadrons here
obs2.onDetection( TextOutput('photon_output.txt', Output.PhotonOutput1D) )
m.add(obs2)
```

Reply:
Yes, you're right. Using ObserverDetectAll would probably be clearer.